README.md
![Screenshot](https://github.com/fu8uk1/fu-b0x/blob/master/pictures/fu-b0x.png)


```
 + Autor: FU8UK1
```
### WARNING
```
DEVELOPERS ASSUME NO LIABILITY AND ARE NOT       
RESPONSIBLE FOR ANY MISUSE OR DAMAGE CAUSED BY  
THIS PROGRAM                                    

```

![Screenshot](https://github.com/fu8uk1/fu-b0x/blob/master/pictures/fu8uk1.png)

<h1 align="center"> |Changing the Description of this tool  |<br>
 |Won't made you the coder ^_^ !!!      |<br>
 |#Respect Coderz ^_^                   | </h1>




### DESCRIPTION
```
21 TOOLS IN 1
```
LAST UPDATE 08/07/2023


### REQUIREMENTS

<strong>*****install ruby*****</strong>


$ sudo apt install ruby


<strong>*****install lolcat*****</strong>

$ wget https://github.com/busyloop/lolcat/archive/master.zip

$ unzip master.zip

$ cd lolcat-master/bin

$ gem install lolcat


<strong>*****install git*****</strong>

$ sudo apt install git-all


### INSTALL fu-b0x

$ git clone http://github.com/fu8uk1/fu-b0x

$ cd fu-b0x

$ chmod +x fu-b0x.sh

$ bash ./fu-b0x.sh


![Screenshot](https://github.com/fu8uk1/fu-b0x/blob/master/pictures/captu1.png)

![Screenshot](https://github.com/fu8uk1/fu-b0x/blob/master/pictures/captu2.png)




changelog.md

\- **Version 7**
\* 08/07/2023
\*Ajout et suppressions de plusieurs logiciels
\*fixed 19 bugs
\*Rajout des trad en français sur le soft




\- **Version 6**
\* 12/11/2021
\*Added mariana-trench soft
\*fixed 2 bugs



\- **Version 5**
\* 09/06/2021
\*remove all software except 20
\*in addition to downloading the software, fu-b0x also installs them
\*fixed 4 bugs



\- **Version 4.4** 

\* 27.09.2020

\* Deleted 4 softs : CoD hacks , Bw hacks , TTr Tools , CoD4 Hacks , Litetools






\- **Version 4.3** 

\* 13.09.2020

\* Added description for install lolcat

\* Deleted 7 softs : ContexPloit , Firefox-Plugin-Pop , First Bitcoin Miner , Mesos Bitcoin , Admin Pedral , QJDID , QFloodSms

\* Added 7 softs : Malware Jail , Gitminer , FoolAv , Gophish , Arpy , Mara , APKTool





\- **Version 4.2** 

\* 06.09.2020

\* Added option "a" for install all softs 

\* Deleted 8 softs : MrCakil , Mr Rv1 , Termux-Banner , Plutus , Hacking Tools , Fbht , Overload Dos , Awesome Hacking 





\- **Version 4.1** 

\* Deleted 32 soft : Bitcoin-Wallet , Camstream , XEIT-Cyber , CSRfpocmaker , PHP-Backconnector , Tool-Kit , Myenc , Blacktrack , Blackmail , BlackNmap , WP-Hunter , Remote-Shell , Killr , Nettacker , fucking-rat , Bruteforce Gmail , Lhst , PhishingGame , Ipddos , MultiSpam , Wascan , Jwt-Cracker , Md-Password , Watweb , Spamchat , Vulnscanner , IGP , Stagefight , Pencari , Galaulers , Typeracer , Debiantools 

\* Added 19 soft : Andorat , Telekiller , Rat Via Telegram , Eggshell , LiteHTTP , Sitadel , Kalitorify , BruteX , Veil , Galileo , Fbi , TigerShark , Sms And Email Bomber , Sifter , Torat , ScanQli , Cynet , Ezsploit , WhatsappHack , Trojan-MDREO

\* Added all descriptions of softs





\- **Version 4.0** 

\* Deleted soft : Bom Sms 3 , Visql , Gadogado , CnkSpam , KtlkKgenerate , A-rat , XLpy , BotFbBongDjon , RusSpam , Anonymous , T-DYF , Zeus-Bot , Gabutz , 4wsectools , Mr.rv.1 , Mr.rv.2 , HPA51369 , PemulungBTC , TouchURL , IPTrack , Kuyang-Tool , Email-Bomber , Ip-Gathering , Scorpion , NewSpammer , Komodo , HN-Installer , Deface-Create , Good-Termina , hostcheker , Webkit , Spam-Mantan , Genscript , AVARSpam , Zones ,SKLOKmed , NetKiller , XPL-Search , Hashzer , TuyulBtn , 1337hash , Cyberscan 

\* Added soft : PyLoggy , Osx Script , Torghost , Evil-Droid , Fuxsocy , Sylver , 4nonimizer , Facebash , Horn3t , SharpAttack , Cr3d0v3r , Poseidon-Bot , Pentest-Tools-Framework , Photon , SourceLeakHacker , RapidScan , Operative-Framework , Delta , Iot-Implant-Toolkit , Proxy-List , TorrGrab , BlackWidow , SecLists , Pixload , Unfx-Proxy-Checker , IP-Tracer , RusseLogger , Rat-Hodin , TalentRat , Saycheese , HiddenEye , PRETty , Shellpy , Buster , SocialFish , Blazy , Instagram-Toolbox , Crawlbox , Black-Dorker , Wordpress-Hacker , EvilOSX , PhoneInfoga

\* Added 189 descriptions of softs 





\- **Version 3.02**  

\* Deleted soft : Nmap , Udfhack , Mirai , Bom , Tool , Malware , IFC , Kawai Botnet , Darksploit , Rootnet

\* Added soft : Shellphish , Vulnx , Email2PhoneNumber , Seeker , Inshackle , Gentiny , PyFlooder , TBomb , Spy-King , LittleBrother

\* Deleted file "install.sh" because problems with compatibility 





\- **Version 3.01**  

\* Add install.sh for requirements

\* Added soft : Cam-Hacker 

\* Add files changelog.md



\- **Version 3.0**  
\* Initial Release


install.sh
#!/bin/bash
#Author: github.com/fu8uk1
#Instagram: @fu8uk1
trap 'echo exiting cleanly...; exit 1;' SIGINT SIGTSTP

checkroot() {

if [[ "$(id -u)" -ne 0 ]]; then
   printf "\e[1;77mPlease, run this program as root!\n\e[0m"
   exit 1
fi

}

checkroot

(trap '' SIGINT SIGTSTP && command -v python > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling python, please wait...\n\e[0m"; apt update > /dev/null && apt -y install python-pip > /dev/null || printf "\e[1;91mPython Not installed.\n\e[0m"; }) & wait $!

(trap '' SIGINT SIGTSTP && command -v python3 > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling python3, please wait...\n\e[0m"; apt update > /dev/null && apt -y install python3-pip > /dev/null || printf "\e[1;91mpython3 Not installed.\n\e[0m"; }) & wait $! 

(trap '' SIGINT SIGTSTP && command -v php > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling php, please wait...\n\e[0m"; apt update > /dev/null && apt -y install lsb-release apt-transport-https ca-certificates > /dev/null || printf "\e[1;91mPhp Not installed.\n\e[0m"; }) & wait $!

(trap '' SIGINT SIGTSTP && command -v php > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling php, please wait...\n\e[0m"; apt update > /dev/null && apt -y wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg

$ echo "deb https://packages.sury.org/php/ buster main" | sudo tee /etc/apt/sources.list.d/php.list > /dev/null || printf "\e[1;91mPhp Not installed.\n\e[0m"; }) & wait $!

(trap '' SIGINT SIGTSTP && command -v php > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling php, please wait...\n\e[0m"; apt update > /dev/null && apt -y install php7.4 > /dev/null || printf "\e[1;91mPhp Not installed.\n\e[0m"; }) & wait $!

(trap '' SIGINT SIGTSTP && command -v lolcat > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling lolcat, please wait...\n\e[0m"; apt update > /dev/null && apt -y pip2 install lolcat > /dev/null || printf "\e[1;91mLolcat Not installed.\n\e[0m"; }) & wait $!

(trap '' SIGINT SIGTSTP && command -v Git > /dev/null 2>&1 || { printf >&2  "\e[1;92mInstalling git, please wait...\n\e[0m"; apt update > /dev/null && apt -y install git-all > /dev/null || printf "\e[1;91mgit Not installed.\n\e[0m"; }) & wait $!



printf "\e[1;92mAll Requires are installed!\n\e[0m"


fu-b0x.sh
#!/bin/bash
#version 7

#fu8uk1

#Changing the Description of this tool |
#|Won't made you the coder ^_^ !!!      |
#|#Respect Coderz ^_^                   | 


#08/07/2023

clear
# Variables
b='\033[1m'
u='\033[4m'
bl='\E[30m'
r='\E[31m'
g='\E[32m'
bu='\E[34m'
m='\E[35m'
c='\E[36m'
w='\E[37m'
endc='\E[0m'
enda='\033[0m'
blue='\e[1;34m'
cyan='\e[1;36m'
red='\e[1;31m'

figlet FU-B0X  | lolcat

echo -b "_____________________________________________________________" | lolcat
echo -b "TYPE      : BASH $green " |lolcat
echo -b "VERSION   : 7 " | lolcat
echo -b "TOTALS    : 21 TOOLS " | lolcat
echo -b "AUTHOR    : FU8UK1 AKA WOLFARCH$green " |lolcat
echo -b "ASSOCIATE : FCA $green " |lolcat
echo -b "CONTRIBUTOR : TIGI $green " |lolcat
echo -b "DON'T USE THIS TOOL FOR ILLEGAL ACTIVITIES" | lolcat
echo -b "NE PAS UTILISER CE SCRIPT POUR DES ACTIVITER ILLEGALE" | lolcat
echo -b "Changing the Description of this tool " |lolcat
echo -b "Won't made you the coder ^_^ !!!  " |lolcat
echo -b "Respect Coderz ^_^ " |lolcat
echo -b "Changer la description de ce script" |lolcat
echo -b "Ne fera pas de toi son codeur ^_^ !!!  " |lolcat
echo -b "Respecte les Codeurz ^_^ " |lolcat
echo -b "_____________________________________________________________" | lolcat

sleep 1

###################################################
# CTRL + C
###################################################
trap ctrl_c INT
ctrl_c() {
clear
echo -b $green"[#]> INSTALLATION " |lolcat
echo -b $green"[#]> Thanks For Using My Tools" |lolcat
echo -b $green"[#]> Merci D'Utiliser mon script" |lolcat
sleep 1
echo -b $green"[#]> fu-b0x On Your System" |lolcat

echo -b $green"[#]> See you Again 😎 :)..." |lolcat
figlet FU-B0X | lolcat
sleep 1
exit
}


lagi=1
while [ $lagi -lt 6 ];
do

echo ""
echo ""

echo -e "######################################" | lolcat
echo -e "#LIST OF TOOLS#" | lolcat
echo -e "#LISTE DES LOGICIELS#" | lolcat
echo -e "######################################" | lolcat

echo ""
echo -e "============================" | lolcat
echo -e $b "1. Install shellphish${enda}"
echo -e "Phishing Tool for Instagram, Facebook, Twitter, Snapchat, etc" | lolcat;
echo -e "============================" | lolcat
echo -e $b "2. Install TrEsNaK${endcla}"
echo -e "9 softs in 1" | lolcat;
echo -e "============================" | lolcat
echo -e $b "3. Install Balgo Crypter${enda}"
echo -e " Balgo Crypter est un utilitaire de génération de charge utile de shell inverse codée Xor avec héxadécimal." | lolcat;
echo -e "============================" | lolcat
echo -e $b "4. Install SocialPhish${enda}"
echo -e "The most complete Phishing Tool, with 32 templates +1 customizable" | lolcat;
echo -e "============================" | lolcat
echo -e $b "5. Install Ghost Eye ${enda}"
echo -e "Ghost Eye Informationgathering Footprinting Scanner and Recon Tool Release. Ghost Eye is an Information Gathering Tool I made in python 3" | lolcat;
echo -e "============================" | lolcat
echo -e $b "6. Install maskphish${enda}"
echo -e "MaskPhish is a simple script to hide phishing URL under normal looking URL (google.com or facebook.com)." | lolcat;
echo -e "============================" | lolcat
echo -e $b "7. Install underhanded${enda}"
echo -e "Tool to check if an app is installed on the target device through a link" | lolcat;
echo -e "============================" | lolcat
echo -e $b "8. Install Anon-SMS${enda}"
echo -e "A Tool To Send Messages Anonymously" | lolcat;
echo -e "============================" | lolcat
echo -e $b "9. Install saycheese${enda}"
echo -e "Grab target's webcam shots by link" | lolcat;
echo -e "============================" | lolcat
echo -e $b "10. Install Cam-Hackers${enda}"
echo -e "Hack cameras" | lolcat;
echo -e "============================" | lolcat
echo -e $b "11. Install Sn1per${enda}"
echo -e "Automated pentest framework for offensive security experts" | lolcat;
echo -e "============================" | lolcat
echo -e $b "12. Install SearchCAM${enda}"
echo -e "txtool is made to help you for easly pentesting in termux" | lolcat;
echo -e "============================" | lolcat
echo -e $b "13. Install inshackle-bot${enda}"
echo -e "Get unlimited followers on Instagram for free" | lolcat;
echo -e "============================" | lolcat
echo -e $b "14. Install Storm-Breaker${enda}"
echo -e "Tool social engineering [Access Webcam & Microphone & Os Password Grabber & Location Finder] With Ngrok" | lolcat;
echo -e "============================" | lolcat
echo -e $b "15. Install Ufonet${enda}"
echo -e "Denial of Service Toolkit" | lolcat;
echo -e "============================" | lolcat
echo -e $b "16. Install PyFlooder${enda}"
echo -e "A http flood python script that could stop a normal website in 10s" | lolcat;
echo -e "============================" | lolcat
echo -e $b "17. Install Gentiny${enda}"
echo -e "URL shortener" | lolcat;
echo -e "============================" | lolcat
echo -e $b "18. Install Evil-Droid${enda}"
echo -e "Evil-Droid is a framework that create & generate & embed apk payload to penetrate android platforms" | lolcat;
echo -e "============================" | lolcat
echo -e $b "19. Install Seeker${enda}"
echo -e "Accurately Locate Smartphones using Social Engineering" | lolcat;
echo -e "============================" | lolcat
echo -e $b "20. Install Facebash${enda}"
echo -e "Tool To Brute Force Facebook Using TOR" | lolcat;
echo -e "============================" | lolcat
echo -e $b "21. Install mariana-trench${enda}"
echo -e "Our security focused static analysis tool for Android and Java applications." | lolcat;
echo -e "============================" | lolcat
echo -e $b "a. Download all softs${enda}"
echo -e "Download all softs in one time" | lolcat;
echo -e "============================" | lolcat
echo -e $b "00. Exit${enda}";
echo -e "============================" | lolcat
echo -e "╭─[CHOOSE THE NUMBER]"
read -p "   ╰─root@./fu-b0x=" pil;

#Install shellphish

case $pil in
1) git clone https://github.com/fu8uk1/shellphish
echo -e "${y} Installing shellphish..."
cd shellphish
bash shellphish.sh
cd

echo

;;

#Install  TrEsNaK

2) git clone https://github.com/fu8uk1/TrEsNaK
echo -e "${y} Installing TrEsNaK..."
cd TrEsNaK
chmod +x TrEsNaK.sh
sudo bash ./TrEsNaK.sh
cd


echo

;;

#Install Balgo Crypter
3) git clone https://github.com/fu8uk1/Balgo-Crypter.git
echo -e "${y} Installing Balgo-Crypter..."
cd Balgo-Crypter
python3 balgo_crypter.py
cd



;;

#Install SocialPhish

4) git clone https://github.com/xHak9x/SocialPhish.git
echo -e "${y} Installing SocialPhish..."
cd SocialPhish
chmod +x socialphish.sh
./socialphish.sh
cd


;;

#Install Ghost Eye

5) git clone https://github.com/BullsEye0/ghost_eye.git
echo -e "${y} Installing Ghost Eye..."
sudo apt install etherape nmap dnsutils gnome-terminal httpie mtr
sudo apt install libgnomeui-0: amd64
cd ghost_eye
pip3 install -r requirements.txt
python3 ghost_eye.py
cd


;;

#Install maskphish

6) git clone git clone https://github.com/fu8uk1/maskphish
echo -e "${y} Installing MASKPHISH..."
cd maskphish
./maskphish
cd


;;

#Install UNDERHANDED

7) git clone git clone https://github.com/fu8uk1/underhanded
echo -e "${y} Installing UNDERHANDED..."
cd underhanded
bash underhanded.sh
cd


;;

#Install Anon-SMS
8) git clone https://github.com/HACK3RY2J/Anon-SMS.git
echo -e "${y} Installing Anon-SMS..."
cd Anon-SMS
sudo bash Run.sh
cd


;;

#Install SAYCHEESE

9) git clone git clone https://github.com/fu8uk1/saycheese
echo -e "${y} Installing SAYCHEESE..."
cd saycheese
bash saycheese
cd


;;

#Install Cam-Hackers

10) git clone https://github.com/AngelSecurityTeam/Cam-Hackers
echo -e "${y} installing Cam-Hackers..."
sudo apt install git
cd Cam-Hackers
pip install -r requirements.txt
python3 cam-hackers.py
cd


;;



#Install Sn1per

11) git clone https://github.com/1N3/Sn1per
echo -e "${y} installing SN1PER..."
cd Sn1per
sudo bash install.sh
cd

;;


#Install SearchCAM

12)git clone https://github.com/AngelSecurityTeam/SearchCAM
echo -e "${y} installing SearchCAM..."
cd SearchCam
pip install -r requirements.txt
python3 searchCAM.py
cd


;;


#Install inshackle-bot

13) git clone https://github.com/fu8uk1/inshackle-bot
echo -e "${y} installing inshackle-bot..."
cd inshackle-bot
bash inshackle.sh
cd


;;

#Install Storm-Breaker

14) git clone https://github.com/ultrasecurity/Storm-Breaker
echo -e "${y} installing Storm-Breaker..."
cd Storm-Breaker
sudo bash linux-installer.sh
python3 -m pip install -r requirments.txt
sudo python3 Storm-Breaker.py
cd


;;


#Install Ufonet

15) git clone https://github.com/epsylon/ufonet
echo -e "${y} installing Ufonet..."
python3 setup.py install
pip3 install GeoIP
pip3 install python-geoip
pip3 install pygeoip
pip3 install requests
pip3 install pycrypto
pip3 install pycurl
pip3 install whois
pip3 install scapy-python3
cd ufonet
bash ./ufonet
cd

;;

#Install PyFlooder

16) git clone https://github.com/fu8uk1/PyFlooder.git
echo -e "${y} installing PYFLOODER..."
cd PyFlooder
python3 pyflooder.py
cd


;;

#Install GenTiny

17) git clone https://github.com/fu8uk1/GenTiny.git
echo -e "${y} installing GENTINY..."
cd GenTiny
pip3 install requests
python3 gentiny2.py
cd


;;

#Install Evil-Droid

18) git clone https://github.com/fu8uk1/Evil-Droid.git
echo -e "${y} installing Evil-Droid..."
cd Evil-Droid
chmod +x evil-droid
./evil-droid
cd


;;

#Install Seeker

19) git clone https://github.com/fu8uk1/Seeker.git
echo -e "${y} installing Seeker..."
cd seeker
chmod 777 install.sh
./install.sh
cd


;;

#Install Facebash

20) git clone https://github.com/fu8uk1/Facebash
echo -e "${y} installing Facebash..."
cd facebash
chmod +x facebash.sh
chmod +x install.sh
sudo ./install.sh
service tor start
sudo ./facebash.sh
cd

;;

#Install mariana-trench

21) https://github.com/fu8uk1/mariana-trench
echo -e "${y} installing mariana-trench..."
cd mariana-trench
cd


;;


# DOWNLOAD ALL

a) git clone https://github.com/fu8uk1/shellphish
git clone https://github.com/fu8uk1/TrEsNaK
git clone https://github.com/fu8uk1/Balgo-Crypter.git
git clone https://github.com/xHak9x/SocialPhish.git

git clone https://github.com/BullsEye0/ghost_eye.git
sudo apt install etherape nmap dnsutils gnome-terminal httpie mtr
sudo apt install libgnomeui-0: amd64
cd ghost_eye
pip3 install -r requirements.txt
cd

git clone https://github.com/fu8uk1/maskphish
git clone https://github.com/fu8uk1/underhanded
git clone https://github.com/HACK3RY2J/Anon-SMS.git
git clone https://github.com/fu8uk1/saycheese

git clone https://github.com/AngelSecurityTeam/Cam-Hackers
sudo apt install git
cd Cam-Hackers
pip install -r requirements.txt
cd

git clone https://github.com/1N3/Sn1per
cd Sn1per
sudo bash install.sh
cd

git clone https://github.com/AngelSecurityTeam/SearchCAM
cd SearchCam
pip install -r requirements.txt
cd

git clone https://github.com/fu8uk1/inshackle-bot

git clone https://github.com/ultrasecurity/Storm-Breaker
 cd Storm-Breaker
sudo bash linux-installer.sh
python3 -m pip install -r requirments.txt
cd

git clone https://github.com/epsylon/ufonet
git clone https://github.com/fu8uk1/PyFlooder.git
git clone https://github.com/fu8uk1/GenTiny.git
git clone https://github.com/fu8uk1/Evil-Droid.git
git clone https://github.com/fu8uk1/Seeker.git
git clone https://github.com/fu8uk1/Facebash
https://github.com/fu8uk1/mariana-trench

echo -e "${y} ALREADY INSTALLED.."

;;



00) echo "AUTHOR:FU8UK1" | lolcat
echo "FU-B0X ON YOUR SYSTEM" | lolcat
echo "FCA ASSOCIATION" | lolcat
echo "THANKS FOR USING MY TOOL" | lolcat
echo "THANKS TO TIGI AND XICO FOR CONTRIBUTIONS" | lolcat
figlet FU-B0X | lolcat
exit
;;

*) echo "Sorry, Your choices its not already [T4T]"
esac
done
done

#[081221215191]
#[GILANG]

